hack/README.md

```diff
@@ -10,7 +10,8 @@ kf.
 - `check-go-sum.sh` Ensures the `go.sum` file is correct.
 - `check-linters.sh` Runs the linters and checks to see if anything is wrong.
 - `clean-integration-tests.sh` Deletes apps that might have been left over
-  from running the integration tests.
+- `create-gke-secret.sh` Creates a GCP ServiceAccount, K8s Secret and updates
```

Comment:
I think the deleted line should be added back to complete the sentence of the line above

Reply:
Ah, good catch.